app/utils/kcidb.py

```diff
@@ -179,6 +179,25 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         'instance': group[models.BOARD_INSTANCE_KEY],
     }
 
+    file_name = "-".join([group[models.NAME_KEY],
+                          group[models.DEVICE_TYPE_KEY]])
+    files = [".".join([file_name, ext]) for ext in ["txt", "html"]]
+
```

Comment:
I'm not crazy about this because it's duplicating the way we generate these filenames from the callback handling code, and if that changes, it has to change in two places.  Do you have any ideas for how to share that code?

Related, in looking at the way we  already create these filenames, it's done based on device-type, and not board-instance, so we already have the problem where log files for two instances of the same device-type in a given lab are going to overwrite each other.  To fix that, I think these log files should be based on the board-instance.

Reply:
c.f. https://github.com/kernelci/kernelci-backend/pull/187

Reply:
But tests are run on a per-device-type basis, not on a per-board basis.  So the same test is never scheduled to run twice in the same lab on different devices, at least that's how things currently stand.

Reply:
OK.

But what about the main question about duplicating the way we construct these filenames and URLs?  It seems a bit fragile to build these paths/urls the same way in two different places, but I'm not familiar enough with the backend structure to know where to best have a shared function for this.

Reply:
How about this (I haven't tried it locally but the log file names should be there):
```python
files = list(group[key] for key in (models.BOOT_LOG_KEY, models.BOOT_LOG_HTML_KEY))
```

Reply:
That's better.  I didn't realize those were already in the group[]  I updated it slightly to use a dict for 'files' also, and then iterate over that for the urls and output_files.